src/sonic-yang-models/tests/yang_model_tests/tests/password_hardening.json

```diff
@@ -0,0 +1,20 @@
+{
+    "PASSWORD_TABLE": {
+        "desc": "Configure policer with all default fields."
+    },
+    "PASSWORD_ENABLE_FEATURE_WITH_POLICIES": {
+        "desc": "Configure password policies and enabled feature."
+    },
+    "PASSWORD_BAD_ENABLE_FEATURE": {
+        "desc": "Configure password policies enabled feature with a wrong value.",
+        "eStrKey" : "InvalidValue"
+    },
+    "PASSWORD_BAD_DIGIT": {
+        "desc": "Configure password Digit Class policy with invalid value",
+        "eStrKey" : "InvalidValue"
+    },
+    "PASSWORD_BAD_HISTORY_CNT": {
+        "desc": "Configure password history_cnt with out of range value",
+        "eStrKey" : "InvalidValue"
+    }
+}
```

Comment:
Please add negative test cases covering len_min, expiration and expiration warning